buildscripts/kokoro/xds.sh

```diff
@@ -22,4 +19,4 @@ python3 grpc/tools/run_tests/run_xds_tests.py \
     --project_id=grpc-testing \
     --gcp_suffix=$(date '+%s') \
     --verbose \
-    --client_cmd='grpc-java/interop-testing/build/install/grpc-interop-testing/bin/xds-test-client --server=xds-experimental:///{service_host}:{service_port} --stats_port={stats_port} --qps={qps}'
+    --client_cmd='grpc-java/interop-testing/build/install/grpc-interop-testing/bin/xds-test-client --server=xds-experimental:///{server_uri} --stats_port={stats_port} --qps={qps}'
```

Comment:
nit: break the line at `--server=` etc

Reply:
Done

Reply:
Oh, I didn't notice that it's inside a quote. Then it's not necessary. My bad.

Reply:
Still looks nicer with the line breaks :)